tests/data/README.md

```diff
@@ -411,6 +411,14 @@ Box structure of the items in this file:
 [alpha] [alpha]
 ```
 
+### File [alpha_noispe.avif](alpha_noispe.avif)
+
+License: [same as libavif](https://github.com/AOMediaCodec/libavif/blob/main/LICENSE)
+
+Source: Same pixels as `color_grid_alpha_nogrid.avif` encoded with
+`avifenc --lossless`. Manually edited with a hex editor to remove the alpha
```

Comment:
Is it necessary to encode ths image losslessly?

Reply:
No, besides making it possible to compare pixel-by-pixel with the other file if it becomes necessary or convenient.